src/config/logging_config.py

```diff
@@ -0,0 +1,26 @@
+import logging
+import sys
+
+import structlog
+
+
+def logger_factory() -> None:
+    logging.basicConfig(format="%(message)s", stream=sys.stdout, level=logging.DEBUG)
```

Comment:
![image](https://github.com/user-attachments/assets/8d671fb4-cbdd-4c42-b532-5bb7f7a81091)

Default log level is info, I don't want to see debug messages as a user